js/views/pauseScreen.js

```diff
@@ -323,5 +390,6 @@ var PauseScreen = createReactClass({
       </div>
     );
   }
-});
+}
+
```

Comment:
Do we also need to enable this behavior on the end screen? I think that from an end-user perspective it might be perceived as being the same or at least a similar state

Reply:
I'll bring this up with Steve.

Reply:
Steve says we should leave the end screen as-is right now.